CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+
+Or maybe you have an idea on how we could reach our [goals](https://cardano-scaling.github.io/cardano-blueprint/#goal) by changing one or the other process? Ideas are collected in the [ideas category](https://github.com/cardano-scaling/cardano-blueprint/discussions/categories/ideas).
+
+We would love to hear about any feedback! 💙
+
+### Report issues
+
+[Submit an issue](https://github.com/cardano-scaling/cardano-blueprint/issues/new) if you spot an errors, or should you encounter a problem with using or contributing to the blueprints themselves.
+
+## Write blueprints
+
+Creating or updating blueprints is the bread & butter of this project and anyone is invited to do just that!
+
+The bandwidth of possible contributions is very broad and can range from fixing typos via the `Suggest an edit` <i class="fa fa-edit"></i> button on the [rendered website](https://cardano-scaling.github.io/cardano-blueprint), over contributing a whole set of documents about how consensus works in Cardano, to including a conformance test suite.
+
+In fact, it's not even set in stone [what a blueprint is](https://cardano-scaling.github.io/cardano-blueprint/#what-is-a-blueprint) and we are gladly exploring any asset that you think covers our values and contributes to reaching the goal of the Cardano Blueprint initiative.
+
+### Creating a pull request
+
+Thank you for contributing your changes by opening a pull request!
+
+We do appreciate it if your pull request meets the following criteria:
+
++ **Description of the changes**: providing a clear summary of the changes is beneficial
++ **Quality of changes**: ensure that new or updated automated tests validate your changes
++ **Scope**: we prefer multiple pull requests that are well-scoped rather than a single large one
++ **Correctness**: the pull request should pass the continuous integration (CI) pipeline.
+
+The project is currently in an expand-and-gather phase with no requirement of approvals, but consider requesting for reviews on Github-suggested reviewers.
+
+Anyone is free to fork the repository and we consider pull requests from forks.
+
+The project is currently situated in the `cardano-scaling` organization, to which we happily invite you and give write permissions onto this repository - so you can pull request from branches directly.
+
+### Review contributions
+
+The Cardano Blueprint aims to be a community effort and we would love to have you become a regular visitor and maybe even reviewer of new contributions! 🤝
+
+## Publishing
+
+Any changes to the `main` branch are automatically published to the live version of the blueprints hosted [via Github Pages](https://cardano-scaling.github.io/cardano-blueprint/). If you have made changes and they are not propagated to the live version, please check the build status on [GitHub Actions](https://github.com/cardano-scaling/cardano-blueprint/actions).
```

Comment:
```suggestion
Any changes to the `main` branch are automatically published to the live version of the blueprints hosted [via Github Pages](https://cardano-scaling.github.io/cardano-blueprint/). If you have made changes and they are not propagated to the live version, please check the build status on [GitHub Actions](https://github.com/cardano-scaling/cardano-blueprint/actions), or make sure you fully reload the webpage (Ctrl+Shift+R on firefox).
```

Reply:
Just as a suggestion. I don't know what it is in other browsers, or whether we actually want to mention that.